man/fread.Rd

```diff
@@ -40,16 +40,12 @@ yaml=FALSE, autostart=NA
   \item{stringsAsFactors}{ Convert all character columns to factors? }
   \item{verbose}{ Be chatty and report timings? }
   \item{skip}{ If 0 (default) start on the first line and from there finds the first row with a consistent number of columns. This automatically avoids irregular header information before the column names row. \code{skip>0} means ignore the first \code{skip} rows manually. \code{skip="string"} searches for \code{"string"} in the file (e.g. a substring of the column names row) and starts on that line (inspired by read.xls in package gdata). }
-  \item{select}{ Vector of column names or numbers to keep, drop the rest. }
+  \item{select}{ A vector of column names or numbers to keep, drop the rest. From v1.12.4, \code{select} may specify types too in the same way as \code{colClasses}; i.e., a vector of \code{colname=type} pairs, or a \code{list} of \code{type=col(s)} pairs. The order that the columns are specified determines the order of the columns in the result. }
```

Comment:
>  The order that the columns are specified determines the order of the columns in the result.

This part has been true, maybe move it before "From v1.12.4"?

Reply:
Good idea, will do.